app/services/evidence/rules/20240425120827_council_tax_payments.rb

```diff
@@ -15,7 +15,6 @@ class CouncilTaxPayments < Rule
         payment.present? && payment.prorated_monthly.to_f > THRESHOLD
       end
 
-      # TODO: Awaiting partner implementation
```

Comment:
I think we need to update the rules though? I'll write a ticket. But just for ref when deleting outdated comments, make sure the action has been taken (which I don't think it has 😬 )

Reply:
It's a joint client and partner question so I don't think there is any further action